api/envoy/api/v2/route/route_components.proto

```diff
@@ -878,6 +886,14 @@ message RouteAction {
   // (e.g.: policies are not merged, most internal one becomes the enforced policy).
   RetryPolicy retry_policy = 9;
 
+  // [#not-implemented-hide:]
+  // Specifies the configuration for retry policy extension. See the supported retry policy
+  // implementations for further documentation. Note that if this is set, it'll take precedence over
+  // the virtual host level retry policy entirely (e.g.: policies are not merged, most internal one
+  // becomes the enforced policy). :ref:`Retry policy <envoy_api_field_route.VirtualHost.retry_policy>`
+  // should not be set if this field is used.
+  google.protobuf.Any retry_policy_extension = 33;
```

Comment:
Ditto. What does "See the supported retry policy implementations for further documentation." mean?

Reply:
Removed. I mean users can refer to retry policy extensions configuration for details since here is just an `Any` typed field.